C/CeresSolver/build_tarballs.jl

```diff
@@ -59,15 +36,22 @@ products = Product[
     LibraryProduct("libceres", :libceres),
 ]
 
+ispowerpc64le(x) = arch(x) == "powerpc64le"
+
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    Dependency("Eigen_jll"),
+    # CeresSolver prefers CXX_THREADS over OpenMP and dependencies for OpenMP are removed
+    BuildDependency("Eigen_jll"; platforms=filter(!ispowerpc64le, platforms)),
```

Comment:
I think OpenMP on PowerPC requires GCC 6: https://github.com/JuliaPackaging/Yggdrasil/pull/5088#issuecomment-1174406071

Reply:
I got the same `error: invalid parameter combination for AltiVec intrinsic` on `powerpc64le` with GCC 6 but no more errors after switching to GCC 8.